packages/expo-updates/ios/EXUpdates/Database/EXUpdatesDatabase.m

```diff
@@ -210,6 +213,35 @@ - (void)markUpdateAccessed:(EXUpdatesUpdate *)update error:(NSError ** _Nullable
   [self _executeSql:updateSql withArgs:@[update.lastAccessed, update.updateId] error:error];
 }
 
+- (void)incrementSuccessfulLaunchCountForUpdate:(EXUpdatesUpdate *)update error:(NSError ** _Nullable)error
```

Comment:
I'm curious why we just don't call `markUpdatesOutdated` where ever this is used instead?

And get rid of the `successful_launch_count` column?